cli/cli_test.go

```diff
@@ -449,8 +515,6 @@ range_max_bytes: 67108864
 	// - attrs: [us-west-1b, ssd]
 	// range_min_bytes: 8388608
 	// range_max_bytes: 67108864
-	// gc:
```

Comment:
oh. sorry, right you are. What I had originally modified was the `zone100 = ...` line, not the example output. Reverted.